src/overlays/actors/ovl_En_Talk/z_en_talk.c

```diff
@@ -30,14 +28,44 @@ const ActorInit En_Talk_InitVars = {
     (ActorFunc)NULL,
 };
 
-#endif
+void EnTalk_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnTalk* this = THIS;
+    s8 targetMode;
+
+    targetMode = this->actor.home.rot.x - 0x1;
+
+    if (targetMode >= 0x0 && targetMode < 0x7) {
+        this->actor.targetMode = targetMode;
+    }
+
+    Actor_SetScale(&this->actor, 1.0f);
+    this->actionFunc = func_80BDE090;
+    this->actor.textId = EN_TALK_GET_TEXT_ID(this);
```

Comment:
```suggestion
    this->actor.textId = EN_TALK_GET_TEXT_ID(&this->actor);
```
With the updated macro.